build.gradle

```
@@ -488,3 +489,10 @@ generateCCDConfig.doLast {
     into layout.buildDirectory.dir('definitions/NFD')
   }
 }
+
+task fortifyScan(type: JavaExec) {
+    main = 'uk.gov.hmcts.fortifyclient.FortifyClientMainApp'
+    classpath = sourceSets.test.runtimeClasspath
```

Thinking it should be += to add it to the existing class path?
```suggestion
    classpath += sourceSets.test.runtimeClasspath
```